doc/configuration/tokens/yubikey.rst

```diff
@@ -30,4 +25,89 @@ Run the command like this::
    privacyidea -U https://your.privacyidea.server -a admin token \
    yubikey_mass_enroll --yubimode YUBICO
 
+This command initializes the token and stores the AES secret and prefix
+in privacyidea, so the toen is immediatly useful. You can choose the slot
+with ``--yubislot`` For further help call
+``privcyidea yubikey_mass_enroll`` with the ``--help`` option.
+
+The second way to enroll a yubikey token is also using ``yubikey_mass_enroll``,
+but with the option ``--filename`` to write to token configuration into the
+specified file.  The resulting file can then be imported into privacyidea:
+Select Tokens -> Import Tokens, Select "OATH CSV" and the file you just created.
+
+Third and last you can use this dialog to enroll a Yubikey AES mode token, if you have
+initialized the yubikey with the external *ykpersonalize* tool.
+
+.. figure:: images/enroll_yubikey.png
+   :width: 500
+
+   *Enroll a Yubikey AES mode token*
+
+
+Remarks about using Yubikey token
+.................................
+
+You can use Yubikey tokens for two more or less distinct applications. The first
```

Comment:
This is an application token. I would rather put this to the application-pam-docs. And maybe reference it here.

There is a doc about privacyidea-pam. 
I would add it there and note: If you are using yubikey you might also use pam_yubico...